boilerplate/immersivewebwg/status-CR.include

```diff
@@ -21,16 +21,7 @@
                A Candidate Recommendation Snapshot has received <a href="https://www.w3.org/2021/Process-20211102/#dfn-wide-review">wide review</a>, is intended to
           gather implementation experience, and has commitments from Working Group members to <a href="https://www.w3.org/Consortium/Patent-Policy/#sec-Requirements">royalty-free licensing</a> for implementations.
         </p>
-        <p>
-            This document was produced by a group operating under the
-            <a id="sotd_patent" property="w3p:patentRules" href="https://www.w3.org/Consortium/Patent-Policy-20170801/">1 August 2017 W3C Patent Policy</a>.
-            <abbr title="World Wide Web Consortium">W3C</abbr> maintains a <a href="https://www.w3.org/groups/wg/immersive-web/ipr" rel="disclosure">public list of any patent
-              disclosures</a> made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent. An individual who has actual knowledge of a patent which the individual believes contains
-            <a href="https://www.w3.org/Consortium/Patent-Policy-20170801/#def-essential">Essential
-            Claim(s)</a> must disclose the information in accordance with
-            <a href="https://www.w3.org/Consortium/Patent-Policy-20170801/#sec-Disclosure">section
-            6 of the <abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy</a>.
-        </p>
+        <p>This document was produced by a group operating under the <a href="https://www.w3.org/Consortium/Patent-Policy/">W3C Patent Policy</a>. W3C maintains a <a rel="disclosure" href="@@URI to IPP status or other page@@">public list of any patent disclosures</a> made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent. An individual who has actual knowledge of a patent which the individual believes contains <a href="https://www.w3.org/Consortium/Patent-Policy/#def-essential">Essential Claim(s)</a> must disclose the information in accordance with <a href="https://www.w3.org/Consortium/Patent-Policy/#sec-Disclosure">section 6 of the W3C Patent Policy</a>.</p>
```

Comment:
You left in a template variable - `@@ URI to IPP status or other page@@`. Same for all the other files.

Also, could you keep the linebreak formatting if possible? It just reduces the diff surface so it's easier to spot changes.

Reply:
Aaaah,, yes. I'll fix these. (Just copied from pubrules template without checking...)
For line break, template does not have one, but let me try to insert.

Reply:
@tabatkins updated via 46ab4d3

original text copied from [pubrules PP section](https://www.w3.org/pubrules/doc/rules?profile=CR#patPolReq)